.pylintrc

```diff
@@ -232,7 +232,7 @@ never-returning-functions=sys.exit
 
 # Format style used to check logging format string. `old` means using %
 # formatting, `new` is for `{}` formatting,and `fstr` is for f-strings.
-logging-format-style=old
+logging-format-style=fstr
```

Comment:
This only works for pylint version 2.4.X.  It is immediately broken in pylint version 2.5.  I'm putting this here for documentation.
 https://github.com/PyCQA/pylint/issues/3570#issuecomment-622980946